my-dsl.md

```diff
@@ -1,38 +1,27 @@
-# Language
-_What is the name of the language? Link the name to its webpage 
-(if appropriate)._
+# Darkroom
 
+## Language
+The language I will be using for my DSL is called Darkroom. You can find it at [darkroom-lang.org](http://darkroom-lang.org/index.html).
 
-# Domain
-_Describe the language's domain in five words._
+## Domain
+Darkroom is a language for creating _image processing pipelines in Terra_ (Terra itself is a low-level programming language).
 
+## Computational model
+Darkroom is actually a very advanced language. While it can efficiently target CPUs, Darkroom can also generate programs for FPGAs and designs for ASICs allowing for extremely fast and efficient image processing. When a program in Darkroom is compiled, the image operations are optimized for the target platform. Actually running Darkroom programs consists of applying the optimized image operations by efficiently loading pixel data into the CPU, FPGA, or ASIC and using CPU instructions or hardware to perform operations from the optimized pipeline. While I don't completely understand how Darkroom works, I know that optimizing how data is loaded and how the image processing pipelines are executed can significantly affect throughput.
 
-# Computational model
-_We don't yet have a great definition of the term "computational model". 
-For now, try to come up with the clearest, most concise explanation of 
-what happens when a program in your DSL runs._
+## DSL-ness
+The Darkroom language really does specialize in image processing pipelines. Darkroom makes it really easy to load and save images and to build pipelines by creating functions that generate output through operations on pixels. While you could theoretically use Darkroom for tasks other than image processing by treating pixels as numbers, it is very clear that this would not be an intended (or very good) use of the language. Darkroom also helps simply image processing operations by encouraging the programmer to work with pixels. Instead of having to define operations on an entire image, you just have to specify how to generate an arbitrary pixel of the output image if given an input image (or images). So, I think it is very clear that Darkroom is a DSL because its operations, functions, and data types revolve around and are specialized for image processing. The syntax also helps convey the task at hand because code isn't cluttered with for loops to iterate over pixels and conditionals to handle edge cases properly.
```

Comment:
Maybe there can be a difference between the way the language pushes you to use it with its syntax, and just considering everything the language can possibly do. If you did this, would that change your answer?